lib/common/formatHelpers/createPropertyFormatter.js

```diff
@@ -19,6 +19,59 @@ const defaultFormatting = {
   suffix: ';'
 }
 
+/**
+ * Split a string comment by newlines and
+ * convert to multi-line comment if necessary
+ * @param {string} to_ret_prop
+ * @param {{comment: string; style: 'short' | 'long'; indentation: string}} options
+ * @param {'short' | 'long'} commentStyle
```

Comment:
I think this param is unnecessary I think?

Reply:
Indentation param also incorrect, and return type was incorrect too, fixed them